### DISCUSSION TITLE
Forwarding already parsed logs

### DISCUSSION THREAD

**Discussion by gcsrellis** [Security-Onion-Solutions/securityonion#4436] [Unsupported Versions] [answered]
Where can I look to forward zeek or ids logs that have already been processed by the preprocessors and such?  I've tried the method in 8.3.5 Forwarding Events to an External Destination but the logs are not in the format I was looking for.  If I could have them forwarded another way that would be helpful.

**Comment by gcsrellis** [comment, marked as answer]
I found the answer in the Docs.  Kept overlooking it

**Reply by b4b857f6ee** [reply]
Hello, i'm trying the same wit the alert or suricata but i can't send anything :/... in fact it's block all...
Even with the doc https://docs.securityonion.net/en/2.3/logstash.html#forwarding-events-to-an-external-destination
